sdcm/tester.py

```diff
@@ -1682,6 +1682,8 @@ def list2reason(exc_list):
             TEST_LOG.error(text)
             TEST_LOG.error("-" * 70)
 
+        self.tag_ami_with_result(error, failure)
```

Comment:
this is not the place to put this. Move to teardown and make sure if it fails it doesn't fail the whole teardown sequence

Reply:
done